src/components/Slider/Slider.tsx

```diff
@@ -1,43 +1,45 @@
 /** @jsx h */
 
 import { h, Component } from 'preact';
-import Rheostat from './Rheostat';
+import Rheostat, { HandleProps } from './Rheostat';
 import cx from 'classnames';
 import { range } from '../../lib/utils';
 import Pit from './Pit';
 import { RangeBoundaries } from '../../connectors/range/connectRange';
+import {
+  RangeSliderCssClasses,
+  RangeSliderWidgetParams,
+} from '../../widgets/range-slider/range-slider';
+import { ComponentCSSClasses } from '../../types';
+
+export type RangeSliderComponentCSSClasses = ComponentCSSClasses<
+  RangeSliderCssClasses
+>;
 
 export type SliderProps = {
   refine(values: RangeBoundaries): void;
   min?: number;
   max?: number;
   values: RangeBoundaries;
   pips?: boolean;
-  step: number;
-  tooltips?:
-    | boolean
-    | {
-        format(value: number): string;
-      };
-  cssClasses: {
-    root: string;
-    disabledRoot: string;
-  };
+  step?: number;
+  tooltips?: RangeSliderWidgetParams['tooltips'];
```

Comment:
are we happy using the widget type here?
Currently widgets depends on component, not the other way around.

Reply:
There's other components where that's the case. It's true it's a little coupled at the moment, but all components for example rely on the widget for the css classes (instead of the other way round), so we can fix that some other time (add a lint rule to avoid depending too circular), fixing this one instance is insignificant.